### PR TITLE
Add more log statement

### PR DESCRIPTION
**Pull request by taikhuu**
Add more log info statement to indicate s3 progress

### CODE REVIEW

File: embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java [hunk 1]
```diff
@@ -201,17 +202,19 @@ private void setHttpProxyInAwsClient(ClientConfiguration clientConfig, HttpProxy
 
     private FileList listFiles(PluginTask task)
     {
+        LOGGER.info("Start listing file with prefix [{}]", task.getPathPrefix());
```
Comment (huylenq) [inline embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java]: Here I am being picky, should this be a "warn" instead? Other than that, LGTM.

Comment (taikhuu) [reply embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java]: I think we are providing information here, not warning user about anything.

Comment (huylenq) [reply embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java]: Oops, silly me, I mean this line https://github.com/embulk/embulk-input-s3/pull/68/files/9398acec7e3391f01d483ef02d5996d13a42d896#diff-6bf6730afed89d21ba77d97065bace32R259.

File: embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java [hunk 2]
```diff
@@ -253,6 +256,7 @@ public static void listS3FilesByPrefix(FileList.Builder builder,
                 if (s.getSize() > 0) {
                     builder.add(s.getKey(), s.getSize());
                     if (!builder.needsMore()) {
+                        LOGGER.info("Too many files matched, stop listing file");
```
Comment (huylenq) [inline embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java]: Over here...

**Pull request merged**